docs/src/guide/loadbalancers.md

```diff
@@ -12,7 +12,7 @@
 :uri-oci-load-balancers: https://docs.oracle.com/en-us/iaas/Content/ContEng/Tasks/contengcreatingloadbalancer.htm#Specifyi
 :uri-oci-oke-flexible-loadbalancers: https://medium.com/@lmukadam/creating-flexible-oci-load-balancers-with-oke-bd98e0318976
 
-== Using Dynamic and Flexible Load Balancers
+## Using Dynamic and Flexible Load Balancers
```

Comment:
Since dynamic is being phased out, we should give instructions for flexible only